### ISSUE TITLE
ödev_5te Main.java yok

### ISSUE THREAD

**Issue by Yigit39** [gusanmaz/NKU_DS_Course_2022#3]
ödev_5 te Main.java eklenmemiş

**Comment by gusanmaz** [comment]
Evet, unutmusum bu odev icin repo'ya sablon kod eklemeyi. Dun (28 Mart( aksama dogru bu odev reposu icin sablon kod ekledim ama odev davetini bu degisiklikten once kabul edenlerin repolarina yansimayacak bu degisiklik tahminimce. Bu durumda https://github.com/gusanmaz/NKU_DS_2022_Java_File2IntArr_Template reposundaki dosyalari kendi Odev5 reponuza ekleyerek odev uzerinde calismaya baslayabilirsiniz.
